content/en/docs/installation/armory-operator/op-troubleshooting.md

```diff
@@ -36,6 +36,17 @@ This error may be normal depending on the frequency of the error. Controllers wo
 }
 ```
 
+## Kustomize `panic` error
+
+Kustomize v4.0+ `kustomize build` throws a `panic` error when it tries to parse and build using the patch files in the `spinnaker-kustomize-patches` repo. For example:
+
+```bash
+panic: runtime error: index out of range [145] with length 145 [recovered]
+ panic: runtime error: index out of range [145] with length 145
+```
+
+Downgrade to Kustomize v3.8.10 to resolve the parsing error. `kubectl` uses Kustomize v2.
```

Comment:
I think we should be a little more explicit here and say that this is for the standalone kustomize and that kubectl comes bundled w/ v2, so they shouldn't run into this error. If they do, something else is causing it.